package.json

```diff
@@ -51,6 +52,7 @@
     ]
   },
   "devDependencies": {
-    "canvas": "^2.6.1"
+    "canvas": "^2.6.1",
+    "miragejs": "^0.1.40"
```

Comment:
let's use fixed versions to avoid surprises.

Reply:
Fixed ! Should I click on the "Resolve conversation button" when all discussions are done or should I let you do it ?
(Really first time contributing on a big project so I don't know how do you manage them 😄)

Reply:
No need to resolve, GitHub automatically handles them when you make new commits so its fine